client/session.go

```diff
@@ -1522,18 +1631,8 @@ func (s *session) FetchBootstrapBlocksFromPeers(
 	// not return an error and if anything goes wrong here we won't report it to
 	// the caller, but metrics and logs are emitted internally. Also note that the
 	// streamAndGroupCollectedBlocksMetadata function is injected.
-	var streamFn streamBlocksMetadataFn
-	switch version {
-	case FetchBlocksMetadataEndpointV1:
-		streamFn = s.streamAndGroupCollectedBlocksMetadata
-	case FetchBlocksMetadataEndpointV2:
-		streamFn = s.streamAndGroupCollectedBlocksMetadataV2
-	// Should never happen
-	default:
-		return nil, errInvalidFetchBlocksMetadataVersion
-	}
-	s.streamBlocksFromPeers(
-		nsMetadata, shard, peers, metadataCh, opts, result, progress, streamFn)
+	s.streamBlocksFromPeers(nsMetadata, shard, peers, metadataCh, opts,
```

Comment:
are you deleting the old codepath in this P.R as well?

Reply:
nevermind I see that you just pushed the switch statement down

Reply:
👍